cpp/src/cylon/table.cpp

```diff
@@ -337,28 +348,32 @@ Status DistributedSort(std::shared_ptr<cylon::Table> &table, int sort_column,
     std::vector<uint32_t> target_partitions, partition_hist;
     std::vector<std::shared_ptr<arrow::Table>> split_tables;
 
-    Status status = MapToSortPartitions(table, sort_column, world_sz, target_partitions,
-                                        partition_hist, sort_options.ascending,
-                                        sort_options.num_samples, sort_options.num_bins);
-    RETURN_CYLON_STATUS_IF_FAILED(status)
+    RETURN_CYLON_STATUS_IF_FAILED(MapToSortPartitions(table, sort_columns[0], world_sz, target_partitions,
```

Comment:
This wouldn't build @chathurawidanage. I changed the macros to require have a semicolon at the end (with my last commit). 
PS: did you take a pull?

Reply:
This should get resolved once this branch is merged with the master.